qemu/tests/luks_convert.py

```diff
@@ -58,3 +58,9 @@ def run(test, params, env):
                                md5_value_to_check=md5_value)
         session.close()
         vm.destroy()
+    logging.debug("Remove the converted img file")
```

Comment:
remove() function will print the proper msg, so we don't need an extra debug msg here